src/lib/icons.tsx

```diff
@@ -139,6 +139,150 @@ export const IconSvgs = {
+    >
+      <rect width="21" height="21" fill="url(#pattern0)" />
+      <defs>
+        <pattern
+          id="pattern0"
+          patternContentUnits="objectBoundingBox"
+          width="1"
+          height="1"
+        >
+          <use xlinkHref="#image0_4176_407" transform="scale(0.01)" />
+        </pattern>
+        <image
+          id="image0_4176_407"
+          width="100"
+          height="100"
+          xlinkHref="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAYAAABw4pVUAAAAAXNSR0IArs4c6QAABxxJREFUeAHtnXnsXUMUx3/WEvwh9i3WCK01/hSJiBAErRb/KEEqIRVia6MlxR80/COEiNgSESqIJRJb2lJUNKE0JJaq9Q+tLSRa20dOcl77e6/3zpu5d+7Mue93b/Ly3p3lzDnf75u7nDkzMzbWHR0CHQIdAh0CHQIdAh0CHQITEAFgK2AycA4wF3gEeBv4CPgS+BnYoB/5LWmSJ2Ue1jpS93CRNQEhrG8ycBBwGbAI+Il4x+/Aa8Ac4Dhgy/rajqgEYH8F6ot4+A+V9A1wB3DoiMIaZhawDXAh8M5Q6JovIJe4mcDWYVaMQGlgWyXis+ZxDm7hK+AqYPsRgNptArAFcAnwQzBM6St8B1wkOrutamkucBSwLD2utVt8CziypbBvrjYwCbgL+Ls2NPkE/AUslEvt5ha2KEWfnN7Nh2P0llcAB7eIgk2qAlP1pS06KpkF/gacv8nSFvwCFmQGLUXzN5mnQt0c96VAw0gb95p929eb99NGgEqpxlPmbvbyL1G/U0ogLLX1nCnHJXC3JXQy6XK/iXsKcEsmACw2Oz8rKcAMi6hk1Ok/GbfJQoq8IAG/ZjTeatO/AAcmJUWfqOSt1eqxHnhFL6fnASeqL20/YGdAvmUQ7GgdqDoJOAuYDdwewfn5XtInL/VNWSTjQ+BSYMc6/1BgSQTjFtbRwbuu/tOsOQrX6eBSlCHZSISIQ/IIb2CrFNTxDGsu9PeBfarYU1YnEiHSyZY2Op6ig0sRenM0ETL0u1MZsFXTIxIihs6sqoezHrAd8H00KOsL+hrYzal0xczIhEggRfxxFODy+hhGkyDP+6dWxHtotciEiNGzhjYaUkCjQ1ZHg7O+oGdD9A8t2wAhErwXL5pFI0TqwxhPwrGhIIeUb4AQsfyCEB2cZQFLw7ArnMpGyGyIkGURVBsbk6i+eH/sKJLm1DFMHgSA44GTHZ+VUTTtFyL3vfrj8cBt/XKznx1ThRB5VwFk3OKfjBYsqKL7xjr6IiiRfFaOP6oMmQJ7G3lkl7jl6oF36nyzQobosXzjvyXgB/CSISOmBKjeXxS42pAhosoT/RoOP1PfmyUzrhyudUkJ4HlLloiXuUTV0mSDI5rPlCrrytCQHhlssXTMc+lclAcstmSATkAK90rrNDJjtnBdEeiuNCM380EcD3PpXJgHTBuUYuA8+Pqb+TG3DLKzC0F3JepkyTKBudKDCFEPdS5dXe3e4MK+MA941CUxU14oITtk0nNYsw8Vgu5KBGSiirVjVAhZ6sK+MA9YZY0NYFQIWVkIuisRWNMR0hgCq13YF+ZFnrAfy7JR6SFrC0F3JepSFbGAjCVnVAhZ78K+MC8hIRI5L9GEPp9JhcqWJKq32kfuYJmmPRQbSlQuT044PzBNhF+5qX05wO6xurNDzrq+Rn1OgG8dAmNmWSPkhJjGlcha48NBXxng0xJhsZOtETIrtoEF8lb1ge1zomGQBbKiJ1kjRBY5aPpY7MNBXxld/KtpxUS+NUJSjC4+2Ae2zwkwLwUbBgn5PIHdc3046CsjqxUkUMxUD5EY3ERrsszoA9vnRNcoTMGJmUsWMCWFwUClASpZ2+rHBApaImR6AnvXVg4FShTkYImQGxMQUj1QHLg+gYIS2XKu5+cQn8ttr4xEnHvK7bX/egJ7r+npF/yd8Jrqi8MoOBcnBxMxvgLwiS9aCcq1nZCPx2Nb6TdwawKgfZtoOyE3VyJhfCWZ2uuLVoJybSek3uWqRwzwZgKwfZpoMyFLenjW/ja0yEybCZlam4ieAI3ztTDps62ECHZxd2aQEByfa0rDZdpKyBW9P3e0b50ancIT6uK0jYTIlOj4CwcIswk9wGWktJGQ6dF6xaAgjeJYXoZWgvS2ESJrsVSfUzhIQNG5uuX/TAB+URNtIkQWUUuziD8wvwitBGltIiR8VLCoB/ikqRc1xxJ/bSFElviLt7aJJykHADLYkvJoAyGyiVn9FRt8SBgso0tTpFwZwTohgsVpgzglPded1VL1EuuEBE9KbYQs4M5EjFgm5J5GwK0iVN9PHkhAilVCZEfR8LnnVcD2raMOyMcbJsUiIY+ZI6NHmvaUJnfXsUaIzG2x1TN6ZIz/Vs/wvw30FiuEyNPU7PE2m/8NnN7Ae4oFQmQV7TPME1CkILAH8GrEnpKbEFnIZt8iW1uTpjd7iaSP4ZDMRYg4CsWGuKN+OVnU7SFertlbchDyhni4c2LXaNsaMFE1+C4lIbKT9bRGwbAiXB4VgTOBDwJ7TApCZEkR2eM9rbfWAjlKjOxs84LnulZNESL7obyou+w0O8JnAXgfHXRdXblxyjiLLDpcdMQmRHbkmQPs5aPjhC0D7Kl7lSwaWASnLiGy3caTwMWylu+EBbiu4cAuwCmhY9Q6Z/BaHbfZta4eXf0OgQ6BDoEOgQ6BDoEOgQ6BtiLwP2F2x3/2Gz8gAAAAAElFTkSuQmCC"
```

Comment:
@jinkang-0 is it better to have the facebook logo as a webp instead of an svg?

Reply:
we can add a separate PR for updating the logos though, it doesn't have to be here.

Reply:
we can add a separate PR to update it though, it doesn't have to be here.

Reply:
I've updated the source of the images on Figma, now they should be proper SVGs. Let's go ahead and update the imports accordingly!